### PR TITLE
Fix colab links

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: tutorials/index.md [hunk 1]
```diff
@@ -16,26 +16,14 @@ Example [Google Colab
...
 
--   **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**
+-   [Case Study: Analyzing Income Distribution](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_income_distribution.ipynb){:target="_blank"}
 
-    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
+-   [Case Study: Prevalence of Obesity in 500 US Cities](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_obesity_prevalence.ipynb){:target="_blank"}
 
--   **Case Study: Analyzing Income Distribution**
+-   [Case Study: Analyzing Genomic Data with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_genomic_data.ipynb){:target="_blank"}
```
Comment (pradh) [inline tutorials/index.md:27]: Not for this PR, but this NB is missing license looks like.

**Review by pradh — APPROVED** [review]
Thanks for the fixes!

File: tutorials/index.md (hunk 1 above)
Comment (beets) [reply tutorials/index.md:27]: good catch!

Comment (beets) [reply tutorials/index.md:27]: i fixed it directly from colab

File: courseware/intro_data_science.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ As an example, MIT's large [Introduction to Machine Learning](https://introml.od
 
 We strongly encourage you to change and adapt these notebooks to fit your needs! You can download any notebook in either .ipynb or .py format by clicking on its link and File > Download.
 
-Datasets can be changed by editing the list of variables queried (see the [“Data Commons for Data Science”](https://colab.research.google.com/drive/1FUQGfoIFr9vwa1wRTmVXVdTYHwr7ntHM) tutorial for more on this); editing framing and questions is as easy as editing text cells.
+Datasets can be changed by editing the list of variables queried (see the [“Data Commons for Data Science”](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb) tutorial for more on this); editing framing and questions is as easy as editing text cells.
```
Comment (pradh) [inline courseware/intro_data_science.md:43]: Sorry for the late comment.  The `{:target="_blank"}` in the other page is nice.  Is it worth adding here?

**Pull request merged**